extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -99,11 +100,19 @@ export class Bind {
     /** @const @private {!../../../src/service/resources-impl.Resources} */
     this.resources_ = resourcesForDoc(ampdoc);
 
+    if (getMode().test) {
+      /**
+       * Array to keep track of mutations during testing
+       * @const @private {!Array<Promise>}
+       */
+      this.mutationPromises_ = [];
```

Comment:
`mutationPromisesForTesting_` would be self-documenting. Sorry should've clarified that in my last comment.